apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.html

```diff
@@ -46,27 +46,38 @@ <h1 class="page-subtitle">{{'GROUP_DETAIL.MEMBERS.TITLE' | translate}}</h1>
     {{'GROUP_DETAIL.MEMBERS.REMOVE_MEMBERS' | translate}}
   </button>
   </span>
-  <button (click)="onListAll()" class="mr-2 mb-2" mat-stroked-button>
-    {{'GROUP_DETAIL.MEMBERS.LIST_ALL' | translate}}
-  </button>
+    <button (click)="onListAll()" class="mr-2" mat-stroked-button>
+      {{'GROUP_DETAIL.MEMBERS.LIST_ALL' | translate}}
```

Comment:
on this button the lable is "list all members" in group and "list member" is label that is in the vo, It would be nice if it same